tests/integration/test_failure_output.py

```diff
@@ -96,10 +95,9 @@ def test_failure_output_after_state_change_for_mk8s(self, single_failing_test_fo
 
         # Setup
         single_failing_test_for_mk8s.tests[0].state = 'passed'  # Change the state of the test to 'passed'
-        single_failing_test_for_mk8s.invoke_zigzag(force_clean_up=False)  # Re-run ZigZag
+        single_failing_test_for_mk8s.assert_invoke_zigzag(force_clean_up=False)  # Re-run ZigZag
         test_runs = single_failing_test_for_mk8s.tests[0].qtest_test_runs
 
         # Test
-        single_failing_test_for_mk8s.assert_queue_job_complete()
```

Comment:
Implemented this for convenience when I want to re-run without cleaning up first. Turns out this pattern is pretty common.